js folder/index.js

```diff
@@ -0,0 +1,65 @@
+// ---------------------- VARIABLES -------------------------- //
+
+const addBtn = document.querySelector('#addBtn');
+const title = document.querySelector('#titleId');
+const author = document.querySelector('#authorId');
+const bookUl = document.querySelector('#books');
+
+// ------------------------ DATA ----------------------------- //
+
+let booksData = [];
+
+// ------------------------ FUNCTIONS ------------------------ //
+function BookConstructor(title, author) {
+  this.title = title;
+  this.author = author;
+}
+
+function loadBooks(index) {
+  // creatingElements
+  const bookLi = document.createElement('li');
+  const titleP = document.createElement('p');
+  const authorP = document.createElement('p');
+  const deleteBtn = document.createElement('button');
+  deleteBtn.setAttribute('id', index);
+  deleteBtn.setAttribute('class', 'buttons');
+  // content inside
+  titleP.textContent = booksData[index].title;
+  authorP.textContent = booksData[index].author;
+  deleteBtn.textContent = 'remove';
+  // appendChilds
+  bookLi.appendChild(titleP);
+  bookLi.appendChild(authorP);
+  bookLi.appendChild(deleteBtn);
+  bookUl.appendChild(bookLi);
+
+  deleteBtn.addEventListener('click', () => {
+    booksData.splice(index, 1);
```

Comment:
[OPTIONAL] Consider using methods that don't mutate the array.

<img width="409" alt="Screen Shot 2022-06-27 at 17 46 01" src="https://user-images.githubusercontent.com/82426682/176049231-038fbe67-5d1d-43a6-b9da-8b18d46f745a.png">